src/main/java/org/fundacion/jala/converter/service/metadata/ExportTypeFile.java

```diff
@@ -74,35 +82,40 @@ private void setDefaultNameFileWithTypeOfFile() {
 
     /**
      * Defines name that the file are exported.
-     * set nameFileExport.
-     * e.g. DragonBall.mp4 --> NameSet.xmp
+     * Set nameFileExport.
+     * e.g. DragonBall.mp4 --> NameSet.xmp.
      */
     private void setNameFileExport(final String newName) {
         this.nameFileExport = newName;
     }
 
     /**
      * Define file are exported with type Txt.
-     * set nameFileCompleteToExport.
+     * Set nameFileCompleteToExport.
```

Comment:
It is "Sets"

Reply:
You are rigth, I have already corrected to third person